device/transport/amqp/test/_amqp_test.js

```diff
@@ -925,7 +898,6 @@ describe('Amqp', function () {
         });
       });
 
-      // skipping until we have the logic in the device client to handle that
       it.skip('forwards errors from the D2C link', function (testCallback) {
```

Comment:
>it.skip [](start = 6, length = 7)

You removed the comment.  Did you mean to also remove the call to .skip?

Reply:
good catch, went back on that. I'm under the impression that we can ignore this error because any transport can reestablish the D2C link when sending the next message so we don't need to "retry" until we have something to send... 

---
In reply to: [145242932](https://github.com/Azure/azure-iot-sdk-node/pull/109#discussion_r145242932) [](ancestors = 145242932)

Reply:
I added a comment to clarify why we're skipping

---
In reply to: [145268291](https://github.com/Azure/azure-iot-sdk-node/pull/109#discussion_r145268291) [](ancestors = 145268291,145242932)